### PR TITLE
go/stackman: Add stackman operator

### PR DESCRIPTION
**Pull request by mslipper**
Converts Stackman into a Kube operator by adding the following:

- The basic Operator scaffolding. Things like RBAC, the manifests for the Operator itself, etc.
- CRDs for the DTL, Sequencer, Gas Oracle, Clique L1, batch submitter, deployer, and actors.

To see how the CRDs get used, see the nightly env in the stacks repo: https://github.com/ethereum-optimism/stacks/tree/master/nightly

Metadata:

- Fixes ENG-1673

### CODE REVIEW

File: go/stackman/Makefile [hunk 1]
```diff
@@ -0,0 +1,201 @@
...
+BUNDLE_DEFAULT_CHANNEL := --default-channel=$(DEFAULT_CHANNEL)
+endif
+BUNDLE_METADATA_OPTS ?= $(BUNDLE_CHANNELS) $(BUNDLE_DEFAULT_CHANNEL)
+
+# IMAGE_TAG_BASE defines the docker.io namespace and part of the image name for remote images.
+# This variable is used to construct full image tags for bundle and catalog images.
+#
+# For example, running 'make bundle-build bundle-push catalog-build catalog-push' will build and push both
+# optimism-stacks.net/stackman-bundle:$VERSION and optimism-stacks.net/stackman-catalog:$VERSION.
+IMAGE_TAG_BASE ?= optimism-stacks.net/stackman
```
Comment (tynes) [inline go/stackman/Makefile:32]: Are you running a docker registry at `optimism-stacks.net`?

**Comment by tynes** [comment]
Perhaps the best way to get this PR merged is through a walkthrough of the code/functionality

File: go/stackman/Makefile (hunk 1 above)
Comment (mslipper) [reply go/stackman/Makefile:32]: No - it gets specified as part of the build command.

**Comment by mslipper** [comment]
Yeah, happy to do a walkthrough. There are also some Notion docs that explain how the overall architecture works, check out the Actor Nightly Testing Overview doc.

**Review by tynes — APPROVED** [review]

**Pull request merged**